fetch.bs

```diff
@@ -3857,8 +3882,13 @@ the request.
    <li><p>Set <var>crossOriginIsolatedCapability</var> to <var>request</var>'s
    <a for=request>client</a>'s
    <a for="environment settings object">cross-origin isolated capability</a>.
+
+   <li><p>Set <var>timingGlobal</var> to <var>request</var>'s <a for=request>client</a>'s
+   <a for="environment settings object">global object</a>.
   </ol>
 
+ <li><p>Assert: <var>timingGlobal</var> is null or a <a for=/>global object</a>.
```

Comment:
I don't think this works. It's passed as "client" or a global object and it's never set to null.

Reply:
Revising to allow "none"